lib/startup.js

```diff
@@ -1,5 +1,10 @@
 if ( Meteor.isClient ) {
-Meteor.startup( function () {
-		$.getScript("https://cdn.rawgit.com/ngzhian/multi-step-modal/master/multi-step-modal.js");
+	Meteor.startup( function () {
+		$.getScript("https://cdn.rawgit.com/ngzhian/multi-step-modal/master/multi-step-modal.js"), function () {
+			sendEvent = function() {
+				$('#offerRequestModal').trigger('next.m.2');
+			}
+			console.log("get overit");
+		}
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.